src/index.js

```diff
@@ -55,15 +68,15 @@ export default function urlJoin(...parts) {
     if (beforePathname) {
         url += beforePathname + (pathname ? '/' : '');
     // Otherwise start with the leading slash
-    } else if (options.leadingSlash) {
+    } else if (options.leadingSlash === true) {
         url += '/';
     }
 
     // Add pathname (foo/bar)
     url += pathname;
 
     // Add trailing slash
-    if (options.trailingSlash && !url.endsWith('/')) {
+    if (options.trailingSlash === true && !url.endsWith('/')) {
```

Comment:
Same here

Reply:
Same reasoning as the comment above. It is fixed now!